addons/analytic/static/src/components/analytic_distribution/analytic_distribution.scss

```
@@ -4,7 +4,8 @@
 
     .analytic_distribution_placeholder {
         height: 1.5em;
-        width: 20px;
+        color: grey;
```

This should be a bootstrap color rather than a default css color. I think `$text-muted` would be the best solution but you can inspect what class/color the other placeholders are using so it looks like the other placeholders.
```suggestion
        color:  $text-muted;
```